tests/Unit/PointwiseFunctions/AnalyticSolutions/Elasticity/Test_HalfSpaceMirror.cpp

```diff
@@ -140,3 +141,18 @@ SPECTRE_TEST_CASE(
         });
   };
 }
+
+// [[OutputRegex, The numerical integral in the AnalyticSolution fails]]
+SPECTRE_TEST_CASE("Unit.AnalyticSolutions.Elasticity.HalfSpaceMirror.Error",
+                  "[PointwiseFunctions][Unit][Elasticity]") {
+  ERROR_TEST();
+  const size_t dim = Elasticity::Solutions::HalfSpaceMirror::volume_dim;
```

Comment:
`const` -> `constexpr` (or just replace the one occurrence of `dim` below with `3` and delete this line)